src/mlpack/methods/hmm/hmm_util_impl.hpp

```diff
@@ -78,18 +78,16 @@ void LoadHMMAndPerformActionHelper(const std::string& modelFile,
   char type;
   ar(CEREAL_NVP(type));
 
-  using namespace mlpack::distribution;
-
   switch (type)
   {
     case HMMType::DiscreteHMM:
       DeserializeHMMAndPerformAction<ActionType, ArchiveType,
-          HMM<DiscreteDistribution>>(ar, x);
+          HMM<mlpack::distribution::DiscreteDistribution>>(ar, x);
```

Comment:
```suggestion
          HMM<distribution::DiscreteDistribution>>(ar, x);
```
I think this could work since you are already in the `mlpack` namespace.  Correct me if I'm wrong...